src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -0,0 +1,69 @@
+package nextstep.subway.line.ui;
+
+import nextstep.subway.line.application.LineService;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.line.dto.LineRequest;
+import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.section.dto.SectionRequest;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class LineController {
+    private LineService lineService;
+    private StationService stationService;
+
+    public LineController(LineService lineService, StationService stationService) {
+        this.lineService = lineService;
+        this.stationService = stationService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        LineResponse lineResponse = lineService.saveLine(lineRequest);
+        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId())).body(lineResponse);
+    }
+    
+    @GetMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> getLines() {
+        List<LineResponse> lineResponses = lineService.findAllLines();
+        return ResponseEntity.ok().body(lineResponses);
+    }
+
+    @GetMapping(value = "/lines/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> getLines(@PathVariable Long id) {
+        LineResponse lineResponse = lineService.findById(id);
+        return ResponseEntity.ok().body(lineResponse);
```

Comment:
```suggestion
        return ResponseEntity.ok(lineResponse);
```

소소하지만..! 다음과 같이 작성할 수 있습니다 ㅎㅎ